pkgs/development/python-modules/deezer-python/default.nix

```diff
@@ -0,0 +1,22 @@
+{ lib, buildPythonPackage, fetchPypi, isPy3k, requests, tornado }:
+
+buildPythonPackage rec {
+  pname = "deezer-python";
+  version = "2.2.2";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "0j3p0ckvkxaq3ng2f13bjhyxxiz0acm7yrsn7x5hmyjy0sk3mjcm";
+  };
+
+  disabled = !isPy3k;
+
+  propagatedBuildInputs = [ requests tornado ];
+
+  meta = with lib; {
+    description = "A friendly Python wrapper around the Deezer API";
+    inherit (src.meta) homepage;
```

Comment:
```suggestion
    homepage = "https://github.com/browniebroke/deezer-python";
```

Reply:
Can you actually start to give a reason for your change requests some day?

Reply:
I am lazy.

I can't click on the inherit when reviewing and need to search/copy the homepage when wanting to lookup more information.